components/forms/index.js

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/prefer-default-export
+export { default as ChangePassword } from './change-password'
```

Comment:
You could use more modern syntax, we have a plugin for it.

```suggestion
export ChangePassword from './change-password'
```